src/sentry/sentry_apps/external_requests/select_requester.py

```diff
@@ -77,25 +75,30 @@ def run(self) -> SelectRequesterResult:
                 extra.update({"url": url})
                 message = "select-requester.request-failed"
 
-            logger.info(message, extra=extra)
-            raise APIError(
-                f"Something went wrong while getting SelectFields from {self.sentry_app.slug}"
+            logger.info(message, exc_info=e, extra=extra)
+            raise SentryAppIntegratorError(
+                message=f"Something went wrong while getting options for Select FormField from {self.sentry_app.slug}",
+                webhook_context={"error_type": message, **extra},
+                status_code=503,
```

Comment:
APIError raises 400

Reply:
We don't really want a 400 here since it's a 3p request failure